### ISSUE TITLE
Atom 1.28.0 won’t open on OS X 10.9.5

### ISSUE THREAD

**Issue by delucis** [atom/atom#17586]
### Description

Atom 1.28.0 won’t open.
1.27.2 opens without errors.

### Steps to Reproduce

1. Open Atom 1.28.0 by any method (including command line, including `atom --safe`, even including `atom --version`)
2. This error window is shown
  ![screen shot 2018-06-26 at 13 28 25](https://user-images.githubusercontent.com/357379/41908654-08c39cf8-7945-11e8-9ea0-be2bd01ee650.png)

3. Most relevant lines from error reporter:
    **Perhaps this is an issue with moving to Electron 2?**
    ```
    Crashed Thread:  0
    
    Exception Type:  EXC_BREAKPOINT (SIGTRAP)
    Exception Codes: 0x0000000000000002, 0x0000000000000000
    
    Application Specific Information:
    dyld: launch, loading dependent libraries
    
    Dyld Error Message:
      Library not loaded: /System/Library/Frameworks/CoreBluetooth.framework/Versions/A/CoreBluetooth
      Referenced from: /Applications/Atom.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
      Reason: image not found
    ```

4. Re-install 1.27.2. Atom opens as expected.

**Expected behavior:** Atom opens (or reports version, etc.)

**Actual behavior:** The system displays the above error message

**Reproduces how often:** 100%

### Versions

- `atom --version` errors, but this is with `1.28.0`
- `apm --version` returns `1.19.0`
- Mac OS X 10.9.5

**Comment by 50Wliu** [comment]
Sounds like the same root cause as #17565.

**Comment by delucis** [comment]
@50Wliu Hadn’t seen that, but I agree!

**Comment by wojiushixiaobai** [comment]
Atom 1.28.0 won’t open on OS X 10.13.5
1.27.2 opens without errors.

**Comment by standurside** [comment]
I'm agree, it happen to me

**Comment by delucis** [comment]
@standurside try redownloading from https://atom.io/

The patch in 1.28.1 worked for me.

**Comment by standurside** [comment]
@ delucis  thanks add
ya, I solved that 1.28.1
